Frontend/app/class/browse/browse.html

```diff
@@ -8,6 +8,19 @@
     </p>
     <br/>
   </div>
+  <div class="col-xs-4" id="joinClassByCode">
+    <h4>Join class by access code</h4>
```

Comment:
Could you add some sort of help text to explain how if a user has a code that they've been given to enter it here to join the class? You can just add it below or if you can do the fancy `fa-info-circle` pop up that occurs when you hover over it that would be cool.